frontend/src/lib/hooks/api/kafkaConnect.ts

```diff
@@ -154,10 +160,13 @@ export function useCreateConnector(clusterName: ClusterName) {
   };
 }
 
-export function useDeleteConnector(props: UseConnectorProps) {
+export function useDeleteConnector(clusterName: ClusterName) {
   const client = useQueryClient();
 
-  return useMutation(() => api.deleteConnector(props), {
-    onSuccess: () => client.invalidateQueries(connectorsKey(props.clusterName)),
-  });
+  return useMutation(
```

Comment:
can we use object syntax here , instead of the function positional stuff , cause that one is deprecated in v5

Reply:
fixed